app/Http/Controllers/Dashboard/IncidentController.php

```diff
@@ -273,6 +289,22 @@ public function editIncidentAction(Incident $incident)
                 null,
                 null
             ));
+
+            // We have an unresolved downtime and the incident gets resolved. Status 4
```

Comment:
Shouldn't it mean "... a unresolved downtime" because the a belongs to downtime and not to unresolved?

Reply:
No it doesn't matter to which word it refers.
An is used for spoken english to avoid having 2 consecutive vowels, which would be hard to pronounce.